test/_requestToken.js

```diff
@@ -71,7 +71,7 @@ describe('_requestToken', function() {
 	it('should fail if request module fails', function(done) {
 		// subsequently, this will test merging options as well
 		AuthClient
-			._requestToken({ method: 'TEST' })
```

Comment:
okay, it was calling a weird method just to make the request module fail, since mocking the request module is not the easiest with how it's set up.

otherwise, looks good to me. will publish today and update the rest module